fpm/after-install.sh

```diff
@@ -42,3 +41,6 @@ grep -q zeyple /etc/postfix/master.cf || add_postfix_master_config
 grep -q zeyple /etc/postfix/main.cf || add_postfix_main_config
 
 postfix reload
+
+# Install python requirements
+pip install gpg || pip install pygpgme
```

Comment:
@DrauzJu Actually this new line is breaking the package for me:

```
[...]
/var/lib/dpkg/info/zeyple.postinst: 46: /var/lib/dpkg/info/zeyple.postinst: pip: not found
/var/lib/dpkg/info/zeyple.postinst: 46: /var/lib/dpkg/info/zeyple.postinst: pip: not found
dpkg: error processing package zeyple (--configure):
 subprocess installed post-installation script returned error exit status 127
Errors were encountered while processing:
 zeyple
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

We should not need to install the pip package since we already install the `python-gpgme` .deb package. Or am I missing something?

Reply:
Ha I see somebody else ran into the same issue: https://github.com/infertux/zeyple/issues/66

Reply:
Yes you're right, sorry for causing the issue, I tested it without the `pip install` and it works fine (https://github.com/DrauzJu/zeyple/commit/20bf051a13de0ef43abdade8d3a88966f29af7e4/checks?check_suite_id=284126438)